pymilvus/exceptions.py

```diff
@@ -207,7 +207,7 @@ class ExceptionsMessage:
     NoPrimaryKey = "Schema must have a primary key field."
     PrimaryKeyNotExist = "Primary field must in dataframe."
     PrimaryKeyOnlyOne = "Primary key field can only be one."
-    PrimaryKeyType = "Primary key type must be DataType.INT64 or DataType.VARCHAR."
```

Comment:
This one can be kept.

Reply:
Maybe next PR, cuz this would make CI fail too. They assert the error message